pyvo/io/uws/endpoint.py

```diff
@@ -108,7 +108,9 @@ def parse(self, iterator, config):
             if start and tag == 'xml':
                 pass
             elif start and tag == 'job':
-                self._version = data.get('version')
+                # Default to v1.0 since the version attribute
+                # was not required until version 1.1.
+                self._version = data.get('version', "1.0")
```

Comment:
Minor stylistic point that single quotes and double quotes are used in the same line of code.

Reply:
I made the requested changes and squashed the commits.